templates/projects/task_scheduler.html

```diff
@@ -91,14 +103,26 @@
         return selectedScheduler() === 'task_queue_scheduler';
     }
 
-    $("#show-customized").on("select2:select", function(evt){
+    function reserveTaskCategorySupported() {
+        return selectedScheduler() === 'task_queue_scheduler';
+    }
+
+    function orderSelect2Items(evt){
         var element = evt.params.data.element;
         var $element = $(element);
         var parentElement = element.parentElement;
 
         $element.detach();
         $(parentElement).append($element);
         $(parentElement).trigger("change");
+    }
+
+    $("#show-customized").on("select2:select", function(evt){
```

Comment:
Perhaps we should not retain the order of the task reservation columns? [co_name, ticker] and [ticker, co_name] should be equivalent as both of them should result in cache
```
co_name:IBM:ticker:IBM_US
```

We should make our cache consistent, right?

Reply:
1. The user experience would be consistent on this page for reserve task columns and customized columns.
2. The task data returned in both cases(retain order or not) would be same.
3. The cache key would be same for all tasks at project level.
In case we wan't to store columns in sorted order just under cache, I believe that could be taken under different ticket/PR.